waku/waku_api/jsonrpc/admin/handlers.nim

```diff
@@ -14,30 +14,13 @@ import
   ../../../waku_relay,
   ../../../waku_node,
   ../../../node/peer_manager,
+  ../../../waku_core,
   ./types
 
 
 logScope:
   topics = "waku node jsonrpc admin_api"
 
-
-
-proc constructMultiaddrStr*(wireaddr: MultiAddress, peerId: PeerId): string =
-  # Constructs a multiaddress with both wire address and p2p identity
-  $wireaddr & "/p2p/" & $peerId
-
-proc constructMultiaddrStr*(peerInfo: PeerInfo): string =
-  # Constructs a multiaddress with both location (wire) address and p2p identity
-  if peerInfo.listenAddrs.len == 0:
-    return ""
-  constructMultiaddrStr(peerInfo.listenAddrs[0], peerInfo.peerId)
-
-proc constructMultiaddrStr*(remotePeerInfo: RemotePeerInfo): string =
-  # Constructs a multiaddress with both location (wire) address and p2p identity
-  if remotePeerInfo.addrs.len == 0:
-    return ""
-  constructMultiaddrStr(remotePeerInfo.addrs[0], remotePeerInfo.peerId)
```

Comment:
Ideally we should make this in a previous and separate PR. But I don't see a big trouble as it is quite clear the movement

Reply:
Thank you. I see it is directly related with this /admin endpoint implementation and would like to avoid code dup.